lib/internal/modules/cjs/loader.js

```diff
@@ -84,13 +84,15 @@ const preserveSymlinksMain = getOptionValue('--preserve-symlinks-main');
 const manifest = getOptionValue('--experimental-policy') ?
   require('internal/process/policy').manifest :
   null;
+const allowedModuleLocations = getOptionValue('--allowed-module-location');
```

Comment:
should cache builtins to avoid mutations from affecting this (comment 1 of n)

```suggestion
const allowedModuleLocations = getOptionValue('--allowed-module-location');
const { relative: PathRelative, isAbsolute: PathIsAbsolute } = path;
```